cmd/wego/add/cmd.go

```diff
@@ -30,9 +30,18 @@ var Cmd = &cobra.Command{
 	Use:   "add [--name <name>] [--url <url>] [--branch <branch>] [--path <path within repository>] [--private-key <keyfile>] <repository directory>",
 	Short: "Add a workload repository to a wego cluster",
 	Long: strings.TrimSpace(dedent.Dedent(`
-        Associates an additional application in a git repository with a wego cluster so that its contents may be managed via GitOps
-    `)),
-	Example: "wego app add .",
+		Associates an additional application in a git repository with a wego cluster so that its contents may be managed via GitOps
+	`)),
+	Example: `
+  # Add podinfo application to wego control from local git repository
+  wego app add .
```

Comment:
I think it's a bit confusing to call out adding the podinfo application without the command mentioning podinfo. Maybe it would be good to add `--name=podinfo`, or just say: "Add application to wego control from local git repository"